### PR TITLE
Allure Report

### PR DESCRIPTION
**Pull request by melnikovmaks**
## Description

Adding Allure Report to tests

### Why?

The addition of Allure Report makes it possible to analyze the test results in detail

## Demo

[How did you test this PR? Include any output/screenshots.]

image

### CODE REVIEW

**Comment by Kirill-Simonovich** [comment]
Add description and images of your Allure report

**Comment by Kirill-Simonovich** [comment]
Where can I see '@Step'?

File: src/test/java/tests/TestClass.java [hunk 1]
```diff
@@ -114,6 +127,8 @@ public void checkAddToWishlist(
     myWishlistsPage.clickDeleteButton();
   }
 
+  @Epic("TESTING FOR http://automationpractice.com/ tasks")
+  @Story("Check the possibility Printed Summer Dress page display for different colour dress")
```
Comment (Kirill-Simonovich) [inline src/test/java/tests/TestClass.java:131]: Why is the name of the test and the story the same?

File: src/test/java/tests/TestClass.java [hunk 2]
```diff
@@ -85,6 +96,8 @@ public void checkWriteReview(
     fadedTShirtPage.clickOkButton();
   }
 
+  @Epic("TESTING FOR http://automationpractice.com/ tasks")
```
Comment (Kirill-Simonovich) [inline src/test/java/tests/TestClass.java:99]: Why @Epic and @Story used for single method?

File: src/main/java/data/model/TestConfigPojo.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  @SerializedName("selenium_timeout")
+  private String mSeleniumTimeout;
+  @SerializedName("selenium_vnc")
+  private String mSeleniumVnc;
+
+  public String getBase() {
+    return mBase;
+  }
+
+  public void setBase(String base) {
```
Comment (Kirill-Simonovich) [inline src/main/java/data/model/TestConfigPojo.java]: why are you writing down all the getters and setters?

File: src/main/java/data/model/TestConfigPojo.java [hunk 4]
```diff
@@ -0,0 +1,113 @@
+
+package data.model;
+
+import javax.annotation.processing.Generated;
+
+import com.google.gson.annotations.SerializedName;
+
+@Generated("net.hexar.json2pojo")
+@SuppressWarnings("unused")
+public class TestConfigPojo {
```
Comment (Kirill-Simonovich) [inline src/main/java/data/model/TestConfigPojo.java]: Where is it used?

Comment (Kirill-Simonovich) [inline src/main/java/data/model/TestConfigPojo.java]: It works?

**Review by Kirill-Simonovich — CHANGES_REQUESTED** [review]

File: src/test/java/tests/TestClass.java (hunk 2 above)
Comment (story) [reply src/test/java/tests/TestClass.java:99]: > Why @Epic and @story used for single method?

?_?

**Pull request merged**